### ISSUE TITLE
Support Symfony 3

### ISSUE THREAD

**Issue by tstoeckler** [jhedstrom/drupalextension#372]
The Symfony dependencies in `composer.json` are currently declared as:
```
    "symfony/dependency-injection": "~2.7",
    "symfony/event-dispatcher": "~2.7"
```
which prevents installing the Drupal extension alongside Symfony 3, for example with latest Drupal 8.4.x.

**Comment by jhedstrom** [comment]
hmm, this should have been fixed years ago in #232...

**Comment by tstoeckler** [comment]
At that time there were no explicit dependencies on Symfony as far as I can tell, so that was just about the drupal-driver compatibility. The dependencies were introduced in https://github.com/jhedstrom/drupalextension/commit/0d658de9740956701dd43e90dc2eae92459dcdac but not sure why exactly.

**Comment by jhedstrom** [comment]
Hopefully this can be resolved in #324.

**Comment by haringsrob** [comment]
Will be closed by #324

**Comment by pfrenssen** [comment]
Closing as duplicate of #324.
